addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -109,6 +109,17 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         }
     }
 
+    .o_wsale_products_grid_before_rail{
+        overflow-y: scroll;
+        scrollbar-width: none;
```

Comment:
Why add the width in the `webkit-scrollbar` then? :thinking: Are both of it necessary?

Reply:
Afair, yes. I will try to remember why and get back to you

Reply:
Oh, it's because different browsers use different tools and without webkit-scrolbar the scroll wouldn't work on some browser.

Reply:
Oh, ok, fine by me then